src/main/java/io/jenkins/plugins/analysis/core/restapi/BlameApi.java

```diff
@@ -0,0 +1,55 @@
+package io.jenkins.plugins.analysis.core.restapi;
+
+import org.kohsuke.stapler.export.Exported;
+import org.kohsuke.stapler.export.ExportedBean;
+
+import io.jenkins.plugins.forensics.blame.FileBlame;
+
+/**
+ * Remote API for a {@link FileBlame}. Simple Java Bean that exposes several methods of an {@link FileBlame} instance.
+ *
+ * @author Kezhi Xiong
+ */
+@ExportedBean
+public class BlameApi {
```

Comment:
Wouldn't it be much simpler if we denormalize the `Blames` and add 3 additional getters to IssueApi? Then `BlameApi` is not required at all?

Reply:
The reason I add the `BlameApi` and to take the blame information as object list in each issue in json/xml is that because I saw the `getLineStart/End` method, so I thought it could be possible that we have multiple blames for an issue which contains multiple lines. Does that situation exist? And about your suggestion, what about if I add some getters like `getAuthor(filename, line)` in `Blames`, or just simply access them in `IssueApi`?

Reply:
> The reason I add the BlameApi and to make the blame infomations as object list in each issue in json/xml is that because I saw the getLineStart/End method, so I thought it could be possible that we have multiple blames for a issue which contains multiple line. Does that situation really exists? And about your suggestion, what about if I add some getters like getAuthor(filename, line) in Blames, or just simplely access them in IssueApi?

No, this situation cannot happen since the blames are computed from the issue `startLine` attribute, so each issue has a 1:1 mapping to the blames currently. It is already denormalized in the model. So it makes sense to just have 3 additional attributes in `IssueApi` that delegate to the corresponding blame, see https://github.com/jenkinsci/warnings-ng-plugin/blob/master/src/main/java/io/jenkins/plugins/analysis/core/model/BlamesModel.java#L78 for details.

Reply:
`BlameApi` is not required at all

Reply:
Thanks for pointing out, I got it.